ocfweb/docs/templates/doc.html

```diff
@@ -4,7 +4,9 @@
 {% block content %}
 <div class="row">
     <div class="col-sm-8 ocf-content-block">
-        {% block before-content %}{% endblock %}
+        {% block before-content %}
+          {% doc_toc toc=toc collapsible=True %}
+        {% endblock %}
```

Comment:
This breaks the margin we add at the top of the page except when the ToC is visible:

![](http://i.fluffy.cc/s182l0FnM1KFgXS0TLSCpGrrkBnh4kr6.png)

It's because the rule that adds it is:
```scss
.ocf-content-block {     
    > :first-child {     
        margin-top: 20px;
    }                    
}                        
```

The rule now targets the `display: none` table of contents, unfortunately.

The best solution I can find is something like
```patch
diff --git a/ocfweb/docs/templates/doc.html b/ocfweb/docs/templates/doc.html
index 3ed35ea..d8499b9 100644
--- a/ocfweb/docs/templates/doc.html
+++ b/ocfweb/docs/templates/doc.html
@@ -4,11 +4,14 @@
 {% block content %}
 <div class="row">
     <div class="col-sm-8 ocf-content-block">
-        {% block before-content %}
-          {% doc_toc toc=toc collapsible=True %}
+        {% block collapsible-toc %}
+            {% doc_toc toc=toc collapsible=True %}
         {% endblock %}
-        {{html|safe}}
-        {% block after-content %}{% endblock %}
+        <div class="ocf-content-block">
+            {% block before-content %}{% endblock %}
+            {{html|safe}}
+            {% block after-content %}{% endblock %}
+        </div>
     </div>
 
     <div class="col-sm-4 ocf-sidebar">
```